sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-resource-server-obo/src/main/java/com/azure/spring/sample/aad/security/AADSampleSecurityConfiguration.java

```diff
@@ -3,21 +3,20 @@
 
 package com.azure.spring.sample.aad.security;
 
+import com.azure.spring.aad.webapi.AADResourceServerWebSecurityConfigurerAdapter;
 import org.springframework.security.config.annotation.method.configuration.EnableGlobalMethodSecurity;
 import org.springframework.security.config.annotation.web.builders.HttpSecurity;
 import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
-import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
-import org.springframework.security.config.annotation.web.configurers.oauth2.server.resource.OAuth2ResourceServerConfigurer;
 
 @EnableWebSecurity
 @EnableGlobalMethodSecurity(securedEnabled = true, prePostEnabled = true)
-public class AADSampleSecurityConfiguration extends WebSecurityConfigurerAdapter {
+public class AADSampleSecurityConfiguration extends AADResourceServerWebSecurityConfigurerAdapter {
```

Comment:
Check [here](https://github.com/Azure/azure-sdk-for-java/blob/4c6215a3cf32ac32204890374f02f32b8d012c5e/sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/aad/webapp/AADWebSecurityConfigurerAdapter.java#L38), perhaps we shouldn't set any requests to need be authenticated in our adapter.